testcases/OsSecureBoot.py

```diff
@@ -0,0 +1,169 @@
+import unittest
+import os, time
+
+import OpTestConfiguration
+
+from common.OpTestSystem import OpSystemState
+from common.OpTestInstallUtil import InstallUtil
+
+"""
+THE PLAN:
+
+ - assert physical presence
+   - clears any existing keys
+   - gets the machine in a known state without secureboot
+ - enroll a set of PK, KEK, db
+   - pregenerated, use secvar sysfs interface
+ - reboot, and ensure secure boot is now enabled
+ - fail to regular kexec an unsigned kernel
+ - fail to load an unsigned kernel
+ - fail to load a dbx'd kernel
+ - successfully load a signed kernel
+ - assert physical presence
+   - ensure machine is in a non-secure boot state
+
+
+"""
+
+# Location of the oskernels.tar and oskeys.tar test data files
+# These belong somewhere else, possibly in the code tree?
+# using a temporary link to the github page, this should probably be something more reliable
+URL = "https://github.com/erichte-ibm/op-test/raw/erichte-ibm/os-secure-boot-squashed/test_binaries"
```

Comment:
I can't think of anything more practical or that is significant better than just hosting the binaries for now (hopefully with good instructions on how those binaries can be reproduced and updated). Will there be a central 'secure & trusted boot' repository / project where those could be hosted? Otherwise we can try to set something up in the op-build repository itself.

Reply:
I'm not sure how we are planning on hosting this project. I'll bring it up with my team, worst case a repo containing static files and documentation will suffice.

Reply:
I'd rather we kept the blobs in the op-test repo and transferred them to the system under test using netcat or something. Alternatively, add a config string for the URL since not everywhere is going to have unfettered access to the external internet.

Reply:
Is there an example of this kind of transfer somewhere in op-test already? I'd prefer that method as well, though I'm also fine with adding a config option if this ends up being too complex.